templates/accounts/resend_activation_mail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <h1 class="mt-2 font-weight-bolder" style="margin-bottom: 30px;">Resend Activation mail</h1>
+        <div class="form-outline">
+            <form method="post">
+                {% csrf_token %}
+                <input type="email" class="form-control" name="email" placeholder="Email"
+                    pattern="[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"
+                    required />
+                <span class="form-error py-3 d-inline-block" style="color:red;">
+                    {{error_message}} </span>
+                <br>
```

Comment:
You should always use margin and padding instead of `br`. So, just remove it from here.